apis/networking/v1beta1/apiproduct_types.go

```diff
@@ -87,6 +89,11 @@ func (a *APISelector) APINamespacedName() types.NamespacedName {
 	return types.NamespacedName{Namespace: a.Namespace, Name: name}
 }
 
+type InfraRateLimitSpec struct {
+	MaxValue int32 `json:"max_value"`
```

Comment:
```suggestion
	MaxValue int32 `json:"maxValue"`
```

Reply:
:+1:  I will update the manifests as well